### PR TITLE
Fio Address List scene themed.

### PR DESCRIPTION
**Pull request by andreyvEze**
fio-names-list

### CODE REVIEW

**Comment by andreyvEze** [comment]
prev PR - https://github.com/EdgeApp/edge-react-gui/pull/2287

**Review by thehobbit85 — APPROVED** [review]

File: ios/edge.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -37,6 +36,7 @@
 		F0D7D6C909584919867CD8B5 /* SourceSansPro-BlackItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 3174EFC33129444580156447 /* SourceSansPro-BlackItalic.ttf */; };
 		F3343A739BF04CF1B73A6AA6 /* SourceSansPro-Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 8203ECA416CB4127BE7B00B3 /* SourceSansPro-Regular.ttf */; };
 		F4A41DF580AB4DA18363D332 /* SourceSansPro-ExtraLight.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 67EDE898A18B445AB08D1ED3 /* SourceSansPro-ExtraLight.ttf */; };
+		FACA1B0225C1989B004CBE73 /* custom.ttf in Resources */ = {isa = PBXBuildFile; fileRef = FACA1B0125C1989B004CBE73 /* custom.ttf */; };
```
Comment (andreyvEze) [inline ios/edge.xcodeproj/project.pbxproj:39]: revert this change

File: ios/edge.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -80,6 +79,7 @@
 		DAF7C41120C505CC005B7898 /* plugins */ = {isa = PBXFileReference; lastKnownFileType = folder; path = plugins; sourceTree = "<group>"; };
 		DCFC9E062F0D18DD4E39B5C5 /* libPods-edge.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = "libPods-edge.a"; sourceTree = BUILT_PRODUCTS_DIR; };
 		E682D9AD200DF9A70042F9BA /* edge.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; name = edge.entitlements; path = edge/edge.entitlements; sourceTree = "<group>"; };
+		FACA1B0125C1989B004CBE73 /* custom.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = custom.ttf; sourceTree = "<group>"; };
```
Comment (andreyvEze) [inline ios/edge.xcodeproj/project.pbxproj:82]: revert this change

File: ios/edge.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -126,7 +126,7 @@
 				6EBB19CA256ABDD4006E5A8F /* Quicksand-Medium.ttf */,
 				6EBB19CC256ABDD4006E5A8F /* Quicksand-Regular.ttf */,
 				6EBB19C9256ABDD4006E5A8F /* Quicksand-SemiBold.ttf */,
-				6EFAA86724F72A18007775E5 /* custom.ttf */,
+				FACA1B0125C1989B004CBE73 /* custom.ttf */,
```
Comment (andreyvEze) [inline ios/edge.xcodeproj/project.pbxproj:129]: revert this change

File: ios/edge.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -290,6 +290,7 @@
 				3D1368BD21DD76BB00DC8CE4 /* edge-core in Resources */,
 				98AF3B3B1F99F7D20083E087 /* audio_sent.mp3 in Resources */,
 				77105E1044684D61B6B0B1EA /* SourceSansPro-Black.ttf in Resources */,
+				FACA1B0225C1989B004CBE73 /* custom.ttf in Resources */,
```
Comment (andreyvEze) [inline ios/edge.xcodeproj/project.pbxproj:293]: revert this change
